### ISSUE TITLE
Usage as a difftool?

### ISSUE THREAD

**Issue by cyberbiont** [L13/vscode-diff#85]
Looks cool, is there a way to use it as a Git difftool?

**Comment by L13** [comment]
Hi and thank you.

Right now there is no such feature. The only way to compare two different commits or branches is to check them out in two different folders. A few days ago I had the same idea, but I realized that it is not as simple as it seems because the ui needs a lot of changes. The idea is still on the table but there is no guarantee that I will do it.

**Comment by cyberbiont** [comment]
Hmm, I presume, it won't work across VS Code instances, if I create a second git worktree and have it open in parallel VS Code instance?

**Comment by L13** [comment]
You need the files "physically" on the disk in two different folders. If you switch the branch in one instance you also switch it in the other. VS Code also does not allow to open the same folder/workspace in two different instances by default. The solution is two different folders. So you have to checkout the project twice.

for example

Folder A
/path/to/workspace/project-name [branch develop]

Folder B
/path/to/workspace/project-name-xyz [branch feature/abc]

This works and is how I do it right now.

**Comment by cyberbiont** [comment]
Yeah, that's exactly what I meant when I mentioned Git worktrees. This Git feature allows you to check the branch into the separate folder on disk (you can simultaneously have several "physical" working directories where different branches are checked). So you can run each of them in separate VS Code instance.
Then you start new diff panel in one of them and set a path to another worktree on the right and then check on both sides whatever commits you want to compare.

**Comment by L13** [comment]
Yes, the extension allows to compare any two folders on your local storage. You can even save the two paths in favorites. The input fields are also supporting predefined variables like ${workspaceFolder} which allows to compare a specific path always to the local instance.

**Comment by L13** [comment]
Hi, after thinking about it I decided this feature is out of scope. Some parts maybe similar but the core would be completely different.
